### DISCUSSION TITLE
GPRM

### DISCUSSION THREAD

**Discussion by zahra-hematy** [community/community#54639] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hi,
I used GPRM to design GitHub and
I created a repository in github and put the codes in the REDME.md file section and no changes were made in my github, my file name in the new repository is readme.
md, I want the front page of my GitHub, which includes repositories, to have the design page, I see these changes on the repository page, but no changes have been applied to the front page of my GitHub, my repo is public and my browser doesn't cache

**Comment by airtower-luna** [comment]
Do you mean that you want the README.md file to show up on your profile? For that you need to name the repository *exactly* like your username, as described [here](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/managing-your-profile-readme#prerequisites). Renaming your existing repository should work.

**Comment by Coderkayc** [comment]
why is the GPRM website not responding? have been trying for week now yet their site isn't responding. pleases is there any alternative?.

**Comment by hsnhacker-CyberDeveloper** [comment]
try this website readme og tiyhubmaker it will work

**Reply by Coderkayc** [reply]
please can you type that again? I didn't quite get that.
